charts/nutanix-csi-storage/questions.yml

```diff
@@ -121,3 +121,10 @@ questions:
     description: "Specify Nutanix Files server name"
     group: "Nutanix Files Settings"
     show_if: "dynamicFileClass=true"
+  - variable: dynamicFileSquashType
+    label: "Squash Type for NFS"
+    type: string
```

Comment:
best here is to give a `type: enum` and to limit the choice to supported option

something like

```
- variable: dynamicFileSquashType
  label: "Squash Type for Dynamic NFS"
  type: enum
  default: "root-squash"
  options: ["root-squash", "all-squash", "none"]
  description: "Specify Nutanix Files Squash Type"
  group: "Nutanix Files Settings"
  show_if: "dynamicFileClass=true"
```

Reply:
Done